src/test/resources/expected/imp.kore.golden

```diff
@@ -3382,27 +3338,11 @@ Definition
                             [ Fix (UnifiedPattern (UnifiedObject (Rotate31 (ApplicationPattern Application
                                 { applicationSymbolOrAlias =
                                     SymbolOrAlias
-                                        { symbolOrAliasConstructor = (Id "strict" AstLocationNone) :: Id Object
+                                        { symbolOrAliasConstructor = (Id "strictAxiom" AstLocationNone) :: Id Object
                                         , symbolOrAliasParams = []
                                         }
                                 , applicationChildren =
-                                    [ Fix (UnifiedPattern (UnifiedObject (Rotate31 (DomainValuePattern DomainValue
-                                        { domainValueSort =
-                                            SortActualSort SortActual
-                                                { sortActualName = (Id "Strict" AstLocationNone) :: Id Object
-                                                , sortActualSorts = []
-                                                }
-                                        , domainValueChild =
-                                            Fix (UnifiedPattern (UnifiedMeta (Rotate31 (StringLiteralPattern (StringLiteral "divAExp")))))
-                                        }))))
-                                    , Fix (UnifiedPattern (UnifiedObject (Rotate31 (ApplicationPattern Application
-                                        { applicationSymbolOrAlias =
-                                            SymbolOrAlias
-                                                { symbolOrAliasConstructor = (Id "firstArgument" AstLocationNone) :: Id Object
-                                                , symbolOrAliasParams = []
-                                                }
-                                        , applicationChildren = []
-                                        }))))
+                                    [ Fix (UnifiedPattern (UnifiedMeta (Rotate31 (StringLiteralPattern (StringLiteral "divAExp,1")))))
```

Comment:
I don't think that this is how we want this to work.

Reply:
?